athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/propertygraph/PropertyGraphHandler.java

```diff
@@ -109,9 +109,15 @@ public void executeQuery(ReadRecordsRequest recordsRequest, final QueryStatusChe
         String labelName = recordsRequest.getTableName().getTableName();
         GeneratedRowWriter.RowWriterBuilder builder = GeneratedRowWriter.newBuilder(recordsRequest.getConstraints());
         String type = recordsRequest.getSchema().getCustomMetadata().get("componenttype");
+        String glabel = recordsRequest.getSchema().getCustomMetadata().get("glabel");
         TableSchemaMetaType tableSchemaMetaType = TableSchemaMetaType.valueOf(type.toUpperCase());
-        
+
         logger.debug("readWithConstraint: schema type is " + tableSchemaMetaType.toString());
+        
+        //AWS Glue converts table name to lowercase, table property 'glabel' stores Amazon Neptune Vertex/Edge labels to be used in Gremlin query
+        if (glabel != null && !glabel.trim().isEmpty()) {
```

Comment:
added this to  handle AWS Glue converting table name to lowercase, the gremlin query fails as Property Graph label names are case sensitive

Reply:
Just making sure: this is something that a customer has to set on their Glue Table, right?

Reply:
Yes, I have updated sample Nodejs script on how to do it